lib/data/external/AwsClient.js

```diff
@@ -327,6 +329,12 @@ class AwsClient {
                   `AWS: ${err.message}`)
                 );
             }
+            if (!completeMpuRes.VersionId) {
```

Comment:
Does versioning have to be enabled on the bucket to use multiple backend routes? If so, I guess I would not expect the complete MPU operation to succeed if also receiving this error. But maybe we want to be flexible here.

Reply:
Versioning has to be enabled on AWS to use multiple backends with AWS. That's why we return an error if no version id is returned (means versioning was suspended at some point)

Reply:
are we doing the same on puts?

Reply:
@LaurenSpiegel yes, we are